polyaxon/templates/api-deployment.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: extensions/v1beta1
```

Comment:
This makes a lot of sense, it's `apps/v1` for all supported k8s version, i.e.`> v1.12`